### PR TITLE
[DNM] General project cleanup

### PR DESCRIPTION
**Pull request by lolletsoc**
@kbairak @glezos

### CODE REVIEW

**Comment by kbairak** [comment]
@lolletsoc is this ready?

**Comment by lolletsoc** [comment]
No, not yet

**Comment by lolletsoc** [comment]
@kbairak Review and merge please! We'll need to do another PR to deal with documentation as well.

File: openformats/utils/__init__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+from ..exceptions import RuleError
+
+_RULES_ATOI = {
+    'zero': 0,
+    'one': 1,
+    'two': 2,
+    'few': 3,
+    'many': 4,
+    'other': 5
+}
```
Comment (kbairak) [inline openformats/utils/__init__.py]: @lolletsoc could it be that some format had different names fo these? Will we be able to override them if that's the case? I don't know how likely this scenario is, but this was the reason I put this as a Handler attribute

Comment (lolletsoc) [inline openformats/utils/__init__.py]: Hmm, fair enough. I'll make a change now.

**Comment by kbairak** [comment]
nice, merging

**Pull request merged**